agent/README.md

```diff
@@ -1,4 +1,65 @@
-# Mona Agent
+# Mona UDS Agent
 
-Mona agent
-Todo(Tzach): Complete this
+### Mona-UDS-Agent docker image
+
+This docker image runs mona-uds-agent, built on top of FluentD Technology.
+
+##### You should use mona-uds-agent when:
+1. You signed up on https://www.monalabs.io/getstarted and have a private endpoint to send your data to Mona
+2. Your models run on a docker orchestration such as Kubernetes and you wish to send their outputs directly to Mona
+3. You want to send your data through Unix Domain Sockets
+
+###### How to run it
+
+The following environment variables are required:
+
+* MONA_USER_ID: your user id (uuid format)
+
+You can use the following YAML formatted file on Kubernetes. 
+UDS agents have to be on all nodes, that's why it's required to use DaemonSet
```

Comment:
They're required to be on "all" nodes, or on all nodes from which data needs to be sent?